apps/web/components/SidebarBox/SidebarBox.tsx

```diff
@@ -1,5 +1,11 @@
 import React, { FC } from 'react'
-import { Box, BoxProps } from '@island.is/island-ui/core'
+import {
+  Box,
+  BoxProps,
+  Stack,
+  Typography,
+  Divider,
```

Comment:
is this being used anywhere?

Reply:
good point, aborted changes to the component and forgot to remove the imports